### PR TITLE
added warning if setTimeRange() is called without a `timeField`

### PR DESCRIPTION
**Pull request by gavinr**
added warning per @jgravois comment https://github.com/Esri/esri-leaflet/issues/1148#issuecomment-479708440. I'm not quite sure if `return [];` is the correct thing to do [here](https://github.com/Esri/esri-leaflet/pull/1152/files#diff-b59294fc1c59924a39c0eb1607d4cc9cR368) ... open to suggestions.

### CODE REVIEW

File: src/Layers/FeatureLayer/FeatureManager.js [hunk 1]
```diff
@@ -361,8 +361,11 @@ export var FeatureManager = VirtualGrid.extend({
       var startTimes = this._startTimeIndex.between(start, end);
       var endTimes = this._endTimeIndex.between(start, end);
       search = startTimes.concat(endTimes);
-    } else {
+    } else if (this._timeIndex) {
       search = this._timeIndex.between(start, end);
+    } else {
+      warn('You must set a timeField to manipulate the start and end time filter.');
```
Comment (jgravois) [inline src/Layers/FeatureLayer/FeatureManager.js]: this looks great to me. good to be even a little _more_ explicit though.

```js
'You must set timeField in the layer constructor in order to manipulate the start and end time filter'
```

**Comment by jgravois** [comment]
after you merge i'd recommend taking a stab at shipping a patch release (to make sure you can).

1. use https://github.com/esri/esri-leaflet/compare/v2.2.4...HEAD to update the CHANGELOG manually

```bash
npm run publish
```

**Comment by gavinr** [comment]
@jgravois did you mean to say `npm run release` (not `npm run publish`) ?

**Comment by jgravois** [comment]
did you mean to say `npm run release`

exactly! 👮

**Pull request merged**